database/migrations/2021_01_21_173609_create_rating_services_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateRatingServicesTable extends Migration
+{
+    const STR_LEN = 256;
```

Comment:
Add to enable icon linking for rating services
`const URL_LEN = 2083;`

Reply:
Did that